### PR TITLE
Fix source code formatting in gateway-ha module

### PR DESCRIPTION
**Pull request by mosabua**
Now checkstyle passes and is enabled by default.

A couple of minor refactoring were necessary. See commits and commit messages.

### CODE REVIEW

File: gateway-ha/src/main/java/io/trino/gateway/ha/GatewayManagedApp.java [hunk 1]
```diff
@@ -4,20 +4,24 @@
 import io.dropwizard.lifecycle.Managed;
 import io.trino.gateway.proxyserver.ProxyServer;
 
-public class GatewayManagedApp implements Managed {
-  @Inject private ProxyServer gateway;
+public class GatewayManagedApp
+        implements Managed
+{
+    @Inject private ProxyServer gateway;
```
Comment (ebyhr) [inline gateway-ha/src/main/java/io/trino/gateway/ha/GatewayManagedApp.java:10]: Wrap after the annotation.

File: gateway-ha/src/main/java/io/trino/gateway/ha/HaGatewayLauncher.java [hunk 2]
```diff
@@ -6,22 +6,27 @@
...
+{
+    public HaGatewayLauncher(String... basePackages)
+    {
+        super(basePackages);
+    }
 
-  public HaGatewayLauncher(String... basePackages) {
-    super(basePackages);
-  }
+    public static void main(String[] args)
```
Comment (ebyhr) [inline gateway-ha/src/main/java/io/trino/gateway/ha/HaGatewayLauncher.java:17]: The original bottom position looks good to me.

File: gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStats.java [hunk 3]
```diff
@@ -1,176 +1,173 @@
...
+    private String clusterId;
+    private String proxyTo;
+    private String externalUrl;
+    private String routingGroup;
+    private Map<String, Integer> userQueuedCount;
+
+    public ClusterStats() {}
+
+    public int getRunningQueryCount()
+    { return this.runningQueryCount; }
```
Comment (ebyhr) [inline gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStats.java:22]: It would be better to format like this:
```java
    {
        return this.runningQueryCount;
    }
```
Same for others.

File: gateway-ha/src/main/java/io/trino/gateway/ha/resource/EntityEditorResource.java [hunk 4]
```diff
@@ -29,150 +29,156 @@
...
 
-    public String getDisplayName()
-    {return this.displayName;}
-
-    public void setDisplayName(String displayName)
-    {this.displayName = displayName;}
-
-    public boolean equals(final Object o)
+    @POST
+    public Response updateEntity(@QueryParam("entityType") String entityTypeStr,
```
Comment (ebyhr) [inline gateway-ha/src/main/java/io/trino/gateway/ha/resource/EntityEditorResource.java:68]: nit: Move the 1st argument to the next line.
```java
    @POST
    public Response updateEntity(
            @QueryParam("entityType") String entityTypeStr,
            @QueryParam("useSchema") String database,
            String jsonPayload)
```
Same for others.

File: gateway-ha/src/main/java/io/trino/gateway/ha/GatewayManagedApp.java (hunk 1 above)
Comment (mosabua) [reply gateway-ha/src/main/java/io/trino/gateway/ha/GatewayManagedApp.java:10]: Done

File: gateway-ha/src/main/java/io/trino/gateway/ha/HaGatewayLauncher.java (hunk 2 above)
Comment (mosabua) [reply gateway-ha/src/main/java/io/trino/gateway/ha/HaGatewayLauncher.java:17]: I am not sure what you mean .. is there anything I should change now?

File: gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStats.java (hunk 3 above)
Comment (mosabua) [reply gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStats.java:22]: ok ... will do for all of them

**Comment by mosabua** [comment]
Comments addressed @ebyhr

File: gateway-ha/src/main/java/io/trino/gateway/ha/resource/EntityEditorResource.java (hunk 4 above)
Comment (mosabua) [reply gateway-ha/src/main/java/io/trino/gateway/ha/resource/EntityEditorResource.java]: Done

File: gateway-ha/src/main/java/io/trino/gateway/ha/HaGatewayLauncher.java (hunk 2 above)
Comment (ebyhr) [reply gateway-ha/src/main/java/io/trino/gateway/ha/HaGatewayLauncher.java:17]: I meant moving this main method to the bottom.

File: gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStats.java (hunk 3 above)
Comment (ebyhr) [reply gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStats.java:22]: This comment isn't yet addressed.

File: gateway-ha/src/test/java/io/trino/gateway/ha/security/TestOidcToken.java [hunk 5]
```diff
@@ -1,46 +1,50 @@
...
-    assertTrue(oidcTokens.getRefreshToken().equals("RefreshTKN"));
-    assertTrue(oidcTokens.getExpiresIn().equals("123456"));
-    assertTrue(oidcTokens.getScope().equals("global"));
-  }
+            oidcTokens = objectMapper.readValue(jsonStr, OidcTokens.class);
+        }
+        catch (JsonProcessingException ex) {
+            log.error(ex.getMessage());
+            fail();
+        }
```
Comment (ebyhr) [inline gateway-ha/src/test/java/io/trino/gateway/ha/security/TestOidcToken.java:42]: Why not remove this catch block?

File: gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStats.java (hunk 3 above)
Comment (mosabua) [reply gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ClusterStats.java:22]: Dang .. I did the whole code base but not this file somehow.. haha. Pushing now

File: gateway-ha/src/test/java/io/trino/gateway/ha/security/TestOidcToken.java (hunk 5 above)
Comment (mosabua) [reply gateway-ha/src/test/java/io/trino/gateway/ha/security/TestOidcToken.java:42]: Because I am only doing a reformat and clean up .. not any behavior change .. just to keep it cleaner and simpler .. clean ups should follow for use though

File: gateway-ha/src/main/java/io/trino/gateway/ha/HaGatewayLauncher.java (hunk 2 above)
Comment (mosabua) [reply gateway-ha/src/main/java/io/trino/gateway/ha/HaGatewayLauncher.java:17]: Intellij reorder I guess .. will move down

File: gateway-ha/src/main/java/io/trino/gateway/ha/security/LbTokenUtil.java [hunk 6]
```diff
@@ -8,7 +8,7 @@
 
 import java.security.interfaces.RSAPublicKey;
 
-public static final class LbTokenUtil
+public final class LbTokenUtil
```
Comment (ebyhr) [inline gateway-ha/src/main/java/io/trino/gateway/ha/security/LbTokenUtil.java:11]: This change is unrelated to "Declare strings as static" commit. Also, I'm not sure why you removed `static`.

File: gateway-ha/src/main/java/io/trino/gateway/ha/security/SessionCookie.java [hunk 7]
```diff
@@ -3,7 +3,7 @@
 import jakarta.ws.rs.core.NewCookie;
 import jakarta.ws.rs.core.Response;
 
-public static final class SessionCookie
+public final class SessionCookie
```
Comment (ebyhr) [inline gateway-ha/src/main/java/io/trino/gateway/ha/security/SessionCookie.java:6]: This change is unrelated to "Declare strings as static" commit. Also, I'm not sure why you removed `static`.

**Review by ebyhr — APPROVED** [review]
